src/reducers/QAReducers.js

```diff
@@ -90,3 +90,15 @@ export const putHelpfulness = (answerID) => {
       console.log(error);
     });
 };
+
+export const putHelpfulnessQuestion = (questionID) => {
+  fetch(`http://3.21.164.220/qa/questions/${questionID}/helpful`, {
+    method: 'PUT',
+  })
+    .then((res) => {
+      console.log('Feedback Received!');
```

Comment:
One random console.log that could be removed

Reply:
@jessepye I removed the console logs. They only would run when clicking 'helpful', but they were unneccesary